.clang-format

```diff
@@ -1,2 +1,16 @@
+# Copyright 2025 Reazon Holdings, Inc.
```

Comment:
I used the above format because I followed the `'Copyright [yyyy] [name of copyright owner]'` format in Apache License, Version 2.0's content.
```
Copyright (c) 2025, Reazon Holdings, Inc.
```